src/seedu/addressbook/data/menu/Price.java

```diff
@@ -10,9 +10,11 @@
 public class Price {
 
     public static final String EXAMPLE = "$4.40";
-    public static final String MESSAGE_PRICE_CONSTRAINTS = "Price must start with a $ sign and must be integer or "
-                                                          + "float in value of 2 decimal places";
-    public static final String PRICE_VALIDATION_REGEX = "\\$[1-9][0-9]*(\\.[0-9]{2})?|\\$0?\\.[0-9]{2}|\\$0";
+    public static final String MESSAGE_PRICE_CONSTRAINTS = "Price must start with a $ sign "
+                                                          + "and must not be more than a 3 digit integer or "
+                                                          + "a float in value of up to 2 decimal places";
+
```

Comment:
The message is too complicated
Should be changed to:
"Price must follow format $A.BC where A is a number of 1-3 digits, B and C are 1 digit each"